File: Regular expressions in Python/step02/2.05/2.5.01.py
'''

Напишите регулярное выражение, которое найдёт все последовательности тест в тексте.

Что нужно найти:

Все последовательности тест

Пример решения:

regex = r'Тут ваше регулярное выражение'

Sample Input 1:

Предлагаю протестировать влияние слова тест в тестовых данных на результаты работы тестируемой программы.

Sample Output 1:

тест тест тест тест

Sample Input 2:

Тут нету этого слова

Sample Output 2:

Sample Input 3:

Снова тестик

Sample Output 3:

тест

'''

regex = r'тест'

File: JavaScript for Beginners/2  Стандартные объекты/2.5.01.js
/*

В этом задании в нашу функцию testDateTime передаются две строки вида "03 November 2017 04:17".
Вам нужно превратить строки в даты, сравнить их. Для той, что больше получить день недели и вернуть его из функции.

Название дня недели должно быть по-русски, с большой буквы, например: "Понедельник".

Sample Input 1:

04 April 1909 17:29
21 July 1909 09:13

Sample Output 1:

Среда

Sample Input 2:

09 May 1909 07:49
25 September 1909 04:13

Sample Output 2:

Суббота

*/

function testDateTime(a, b) {
    var n = new Date(a);
    var m = new Date(b);
    var days = ["Воскресенье", "Понедельник", "Вторник", "Среда", "Четверг", "Пятница", "Суббота"];
    if (n > m) {
        return days[n.getDay()];
    }
    else {
        return days[m.getDay()];
    }
    
}
